lib/addresses.js

```diff
@@ -184,12 +184,28 @@ AddressController.prototype.transformUtxo = function(utxo) {
 AddressController.prototype.multitxs = function(req, res, next) {
   var self = this;
 
-  this.node.getAddressHistory(req.addrs, true, function(err, txinfos) {
+  this.node.getAddressHistoryCount(req.addrs, {}, function(err, count) {
     if(err) {
       return common.handleErrors(err, res);
     }
 
-    res.jsonp(self.transformAddressHistoryForMultiTxs(txinfos));
+    var options = {
+      from: req.query.from || req.body.from || 0,
+      to: req.query.to || req.body.to || count
```

Comment:
I think we could add a condition in getAddressHistory that if a "from" is specified, and not a "to", the "to" will automatically have a default at either the total, or MAX_PAGE_SIZE. Getting the count here also shouldn't be needed, however I do think `getAddressHistoryCount` or similar, is needed for the `?noTxList=1` query.